keyboards/levinson/readme.md

```diff
@@ -3,18 +3,24 @@ Levinson
 
 A split 40% split 4x12 ortholinear keyboard made and sold by Keebio. It's essentially a Let's Split with LED backlight support and 2u thumb key support. [More info at Keebio](https://keeb.io).
 
-Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)  
-Hardware Supported: Pro Micro  
-Hardware Availability: [Keebio](https://keeb.io)  
+Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)
+Hardware Supported: Pro Micro, Elite-C
+Hardware Availability: [Keebio](https://keeb.io)
```

Comment:
@drashna FYI, this line doesn't need two spaces (though it doesn't hurt anything to have them), because the double-line break that follows automatically starts a new paragraph.